### PR TITLE
[FS-22] - Create Price on Stripe

### PR DESCRIPTION
**Pull request by gleandroj**
## Description
[FS-22] - Create Price on Stripe

## Tickets

[FS-22](https://flatirons.atlassian.net/browse/FS-22)

## Screenshots / Demos

[FS-22]: https://flatirons.atlassian.net/browse/FS-22?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: spec/flatirons/saas/services/stripe_service_spec.rb [hunk 1]
```diff
@@ -189,4 +207,107 @@
...
+
+        it 'should create a price' do
+          price = service.create_price(
+            product_id: product.id,
+            unit_amount: unit_amount,
+            currency: currency,
+            recurring_interval: recurring_interval
+          )
+          expect(price).to be
+          expect(price.id).to be
```
Comment (animista01) [inline spec/flatirons/saas/services/stripe_service_spec.rb]: Lets put it as explicitly as we can 
`not_to be_nil`

File: spec/flatirons/saas/services/stripe_service_spec.rb [hunk 2]
```diff
@@ -189,4 +207,107 @@
...
+                                 unit_amount: 4000,
+                                 currency: 'usd',
+                                 product: product.id,
+                               })
+        end
+
+        it 'should list prices to given product' do
+          prices = service.list_prices(product.id)
+          expect(prices).to be
+          expect(prices.length).to eq 2
```
Comment (animista01) [inline spec/flatirons/saas/services/stripe_service_spec.rb]: use `#size`, `length` is more use to count letters in word for instance.

**Review by animista01 — CHANGES_REQUESTED** [review]
does your changes require a swagger doc update? IDK

**Comment by gleandroj** [comment]
> does your changes require a swagger doc update? IDK

It doesn’t

**Review by animista01 — APPROVED** [review]

**Pull request merged**